utils/display.go

```diff
@@ -68,6 +68,52 @@ func PrintEndpointsTable(ends []meroxa.Endpoint) {
 	fmt.Println(EndpointsTable(ends))
 }
 
+func header(h string) string {
+	header := h + "\n"
+	for range header {
+		header += "="
+	}
+	header += "\n"
+
+	return header
+}
+
+func ResourceTable(res *meroxa.Resource) string {
+	mainHeader := header(fmt.Sprintf("Resource %s", res.Name))
+	mainTable := simpletable.New()
+	mainTable.Body.Cells = [][]*simpletable.Cell{
+		{
+			{Align: simpletable.AlignRight, Text: "ID:"},
+			{Text: fmt.Sprintf("%d", res.ID)},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "NAME:"},
+			{Text: res.Name},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "TYPE:"},
+			{Text: res.Type},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "URL:"},
+			{Text: res.URL},
+		},
+		{
+			{Align: simpletable.AlignRight, Text: "Status"},
```

Comment:
Can you rename this to State instead of Status? (for connectors we refer to this field as State (and so does API))

Reply:
Sure, fixed in https://github.com/meroxa/cli/pull/148/commits/df75a278ffa40f0babe2126c5471b22e1151d015

Reply:
@dianadoherty oh, good point. Thanks @owenthereal for fixing it